frontend/src/App.jsx

```diff
@@ -39,6 +42,10 @@ function App() {
     }
   }
 
+  const handleClickLogout = () => {
```

Comment:
Since you already have a button here, can you move the logic from the '/logout' route here? Then we won't need the logout page anymore